api/src/pcapi/routes/backoffice/templates/chronicles/list.html

```diff
@@ -0,0 +1,80 @@
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% from "components/collapsable_text.html" import collapsable_text with context %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Chroniques</h2>
+    <div class="filters">{{ build_filters_form(form, dst) }}</div>
+    <div>
+      {% if rows %}
+        <div class="d-flex justify-content-between">
+          <p class="lead num-results d-inline">{{ rows.total }} résultat{{ rows.total | pluralize }}</p>
+          <div>{% include "components/search/pagination.html" %}</div>
+        </div>
+        <table class="table mb-4">
+          <thead>
+            <tr>
+              <th scope="col"></th>
+              <th scope="col">ID</th>
+              <th scope="col">Titres des œuvres</th>
+              <th scope="col">Contenu</th>
+              <th scope="col">Date de création</th>
+              <th scope="col">Publiée</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for chronicle in rows.items %}
+              <tr>
+                <td>
+                  {% if has_permission("READ_CHRONICLE") %}
+                    <div class="dropdown">
+                      <button type="button"
+                              data-bs-toggle="dropdown"
+                              aria-expanded="false"
+                              class="btn p-0">
+                        <i class="bi bi-three-dots-vertical"></i>
+                      </button>
+                      <ul class="dropdown-menu">
+                        {% if chronicle.isActive %}
+                          <li class="dropdown-item">
+                            <a class="btn btn-sm d-block w-100 text-start px-3"
+                               data-bs-toggle="modal"
+                               data-bs-target="#deactivate-chronicle-modal-{{ chronicle.id }}">Dépublier</a>
+                          </li>
+                        {% else %}
+                          <li class="dropdown-item">
+                            <a class="btn btn-sm d-block w-100 text-start px-3"
+                               data-bs-toggle="modal"
+                               data-bs-target="#activate-chronicle-modal-{{ chronicle.id }}">Publier</a>
+                          </li>
+                        {% endif %}
+                      </ul>
+                    </div>
+                  {% endif %}
+                </td>
+                <td>
+                  <a href="">{{ chronicle.id }}</a>
+                </td>
+                <td>
+                  {% if chronicle.products %}
+                    {% for product in chronicle.products %}{{ product }}{% endfor %}
+                  {% endif %}
+                </td>
+                <td class="pc-mw-50-vw">{{ collapsable_text(chronicle.id, chronicle.short_content, chronicle.content) }}</td>
+                <td>{{ chronicle.dateCreated  | format_date("%d/%m/%Y") }}</td>
```

Comment:
C'est le format par défaut.
```suggestion
                <td>{{ chronicle.dateCreated  | format_date }}</td>
```